tests/integration/workflows/rust_cargo/testdata/hello/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "hello"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+lambda = { git = "https://github.com/awslabs/aws-lambda-rust-runtime" }
```

Comment:
depending on a git rev here is temporary until we can release a new version of the runtime https://github.com/awslabs/aws-lambda-rust-runtime/issues/216

Reply:
Now they have tagged releases, worth changing.